aviary/wren/utils.py

```diff
@@ -114,10 +114,15 @@ def get_aflow_label_aflow(
     _, _, spg_no, *wyks = aflow_label.split("_")
     elems = sorted(el.symbol for el in struct.composition)
     elem_dict = {}
-    subst = r"1\g<1>"  # normalize Wyckoff letters to start with 1 if missing digit
-    for el, wyk in zip(elems, wyks):
-        wyk = re.sub(r"((?<![0-9])[A-z])", subst, wyk)
-        sep_el_wyks = ["".join(g) for _, g in groupby(wyk, str.isalpha)]
+    for el, wyk_letters_per_elem in zip(elems, wyks):
```

Comment:
@janosh is `el` a clear enough name in the short names bad era? Also this `wyks` isn't super clear either